### PR TITLE
Release

### PR DESCRIPTION
**Pull request by jpic**

### CODE REVIEW

File: .openshift/action_hooks/build [hunk 1]
```diff
@@ -0,0 +1 @@
+export OPENSHIFT_PYTHON_WSGI_APPLICATION=memepol/wsgi.py
```
Comment (toabctl) [inline .openshift/action_hooks/build]: Shouldn't that be ```memopol/wsgi.py``` ?

File: .travis.yml [hunk 2]
```diff
@@ -1,16 +1,57 @@
...
+- '2.7'
+before_install:
+- pip install codecov
 install:
-  - pip install django
-  - pip install -U pip
-  - pip install -r requirements.txt
-  - cp memopol/config.json.sample memopol/config.json
+- pip install -e .
+- pip install flake8 pep8 pytest pytest-django pytest-cov codecov
```
Comment (toabctl) [inline .travis.yml]: would be nice to have this in a test-requirements.txt file (similar to requirements.txt)

File: .travis.yml [hunk 3]
```diff
@@ -1,16 +1,57 @@
...
 before_script:
-  - npm install -g bower
-  - bower install
-  - npm install
+- bin/install_client_deps.sh
 script:
-  - ./manage.py migrate
-  - node_modules/gulp/bin/gulp.js less
+- bash -c '! lesscpy -N static/less/base.less 2>&1 | grep Error'
+- pep8 . --exclude '*/migrations,docs,static' --ignore E128
```
Comment (toabctl) [inline .travis.yml]: Isn't flake8 already calling pep8?

File: bin/lib.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
+if [ -n "$OPENSHIFT_HOMEDIR" ]; then
```
Comment (toabctl) [inline bin/lib.sh]: What about a shebang (```#!/bin/bash```) ?

Comment (jpic) [inline bin/lib.sh]: This script is only meant to be sourced, executing it wouldn't do anything, it doesn't have the executable bit so a shebang wouldn't do anything here.

File: .openshift/action_hooks/build (hunk 1 above)
Comment (jpic) [inline .openshift/action_hooks/build]: That's probably why it wasn't working and I still had to pass it on the command line to create the app, thanksssssssssssss

File: .travis.yml (hunk 3 above)
Comment (jpic) [inline .travis.yml]: I'm sure it does now, but wasn't at the time of writting.

File: .travis.yml (hunk 2 above)
Comment (jpic) [inline .travis.yml]: Maybe it would be nicer to add them as extra_requires in setup.py and do pip install -e .[testing], would that work too ?

**Pull request merged**